service/service.py

```diff
@@ -258,6 +258,34 @@ def update_stock(product_id, condition, operation, amount):
     app.logger.info(msg_tmp+"%d items having product_id %d and condition %s.", amount, product_id, condition)
     return make_response(jsonify(inventory.serialize()), status.HTTP_200_OK)
 
+################################################################################
+# UPDATE AN EXISTING PRODUCT'S AVAILABILITY
+################################################################################
+@app.route("/inventory/<int:product_id>/<string:condition>/<int:available>", methods=["PUT"])
+def update_stock(product_id, condition, available):
```

Comment:
@kainattnaeem, be sure to change the name of the function when you replicate it :)

Reply:
sorry about that, fixing it